stable/app/values.yaml

```diff
@@ -142,4 +146,23 @@ telegraf:
   command:
     - telegraf
 
-podLabels: {}
\ No newline at end of file
+otelcollector:
```

Comment:
let's update the README.md

Reply:
why do we document again on readme. i see that they are not description of each config. it just has default values and type.. which is what `values.yaml` is for. i feel like that approach has more cons than pros (pros: none?, cons: easy possibility of drift).

Reply:
it provides better readability, we could use [helm-docs](https://github.com/norwoodj/helm-docs) to autogenerate it

Reply:
correctness is more important though. if you check the readme and compare to this right now, it is already out-of-sync for many configs. even if the configs were actually in-sync, i could never be sure. that doubt itself is enough to compel me to never look at the readme and always rely on the `values.yaml` honestly.

Each config can be documented inside the values.yml file using comments -- this i think is the more common approach as well. For example: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/values.yaml

And this might be personal opinion: but i feel for nested configs like this, yaml is more readable because sub-configs are nested and grouped together whereas a flat list has no such organisation and i have to mentally group them together.

Reply:
yup agree on this, let's do that,

any thought @mabdh @AkarshSatija ?

Reply:
+1 for keeping the README.md only for the important things (backward compatibility, etc) and just leave each detail documentation in `values.yaml` itself.